custom/opm/reports.py

```diff
@@ -579,6 +579,11 @@ def get_rows(self):
     @property
     @memoized
     def cases(self):
+        if 'debug_case' in self.request.GET:
+            case = CommCareCase.get(self.request.GET['debug_case'])
+            assert case.domain == DOMAIN
```

Comment:
it's a good general practice to not rely on asserts for control flow, so it would be preferable to do the domain check and raise an exception instead